source/tests/endpoints/devices/test_devices.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import json
+
+from endpoints.devices.business_logic.Devices import BusinessLogicDevices
+from model.Database import Database
+from model.PluginManager import PluginManager
+from plugins.mock.light.Light import Light
+from plugins.mock.lock.Lock import Lock
+
+
+class TestDevices(unittest.TestCase):
+
+    def setUp(self):
+        self._database = Database()
+        self._plugin_manager = PluginManager()
+        self._logic = BusinessLogicDevices(self._database, self._plugin_manager)
+
+    def test_database_set_correctly(self):
+        self.assertEqual(self._database, self._logic._database)
+
+    def test_plugin_manager_set_correctly(self):
+        self.assertEqual(self._plugin_manager, self._logic._plugin_manager)
+
+    @unittest.skip("Currently not working because of singleton")
```

Comment:
Me and Suzanne had a discussion about why we need a singleton for the database. Since we declare the database currently in one place there isn't really a reason for having it as a singleton. Suzanne changed this is her testing module. To prevent redoing these tests i just skipped them for now.